suite/tests/CMakeLists.txt

```diff
@@ -4674,6 +4674,17 @@ if (BUILD_CLIENTS)
       # it checks encodings).
       "opcode_mix")
 
+    if (X86 AND X64 AND ZLIB_FOUND)
+        set(testname "tool.record_filter_encodings2regdeps")
+        torun_record_filter("${testname}" ${ci_shared_app}
+          "record_filter_encodings2regdeps"
+          # We assume the app name starts with "s" here to avoid colliding with
+          # our output dir, while still letting the single precmd remove both.
+          "${drcachesim_path}@-simulator_type@record_filter@-encodings2regdeps@-indir@${testname}.s*.dir/trace@-core_sharded@-cores@4@-outdir@${testname}.filtered.dir"
+          # We run the opcode_mix analyzer to test econdings2regdeps filtered traces.
+          "opcode_mix")
```

Comment:
Is this the right way to run "opcode_mix" on a trace filtered by record_filter (with encodings2regdeps)?

Reply:
Looks like it.  I don't think it needs the quotes.

Reply:
Yes, it works fine now.